### PR TITLE
Add Installation with conda

### PR DESCRIPTION
**Pull request by DunklesArchipel**
Added installation instructions to create a conda/mamba environment.

The packages that have to be installed are reduced to a minimum. You might have to add some that I missed, which could introduce some bugs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,51 @@
...
+
+Download the recent version of the database with the [unitpackage](https://echemdb.github.io/unitpackage/) API. Rerun the following lines, when the database changes.
+
+```sh
+python
 from unitpackage.collection import Collection
-db = Collection.from_remote()
+db = Collection.from_remote() # the download may take a few minutes
 db.save_entries('entries')
+exit()
```
Comment (DenizCankurtaran) [inline README.md:42]: We also have the file `db/_persist_db_local.py`, which could be executed instead. Haven't maintained the README yet 😓 

Running `python db/_persist_db_local.py` would accomplish the same thing

File: environment.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+# conda env create -f environment.yaml
+name: echemdb_app
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - numpy>=1.26.4,<2.0.0
+  - pip>=24
+  - streamlit>=1.36.0,<2.0.0
+  - unitpackage>=0.8.4,<0.9.0
```
Comment (DenizCankurtaran) [inline environment.yaml:12]:
```suggestion
dependencies:
  - numpy>=1.26.4,<2.0.0
  - pip>=24
  - streamlit>=1.36.0,<2.0.0
  - streamlit-extras>=0.4.3,<1.0.0
  - unitpackage>=0.8.4,<0.9.0
```

Hier gibt es ein Problem 😞 Wir benutzen auch die 3rd Party Komponente `streamlit-navigation-bar` ich hab die unter den conda packages nicht gefunden, man müsste also nach dem `conda install` nochmal `streamlit-navigation-bar==3.3.0` ausführen

File: README.md (hunk 1 above)
Comment (DunklesArchipel) [reply README.md:42]: Sure, an approach to download the database without moving into Python is possibly a good idea.

Feel free to include it in this PR or fix it later, when you work more on the documentation.

File: environment.yaml (hunk 2 above)
Comment (DunklesArchipel) [reply environment.yaml:12]: its now included. You can add other relevant things the same way.

**Pull request merged**